plutus-contract/src/Language/Plutus/Contract.hs

```diff
@@ -34,6 +34,7 @@ module Language.Plutus.Contract(
     , WalletAPIError
     , writeTx
     , writeTxSuccess
+    , writeTxConfirmed
```

Comment:
Looking at this, I think `writeTxSuccess` is a confusing name. I had previously got confused and thought it meant "wait until the tx is successfully added to the chain", which it isn't. Especially the contrast between "success" and "confirmed" is weird.

Not totally sure what I'd prefer, I'll think about it.

Reply:
Also, we should probably export `awaitTxConfirmed` too, so people can do it manually if they want.

Reply:
Yeah I named it `writeTxSuccess` because it's a `writeTx` that throws an error instead of returning `Either`. I agree it's confusing

`awaitTxConfirmed` is exported in the "Transactions" section

Reply:
@michaelpj any more thoughts on the naming?

Reply:
It's annoying because we have two dimensions of naming: one for error vs either, one for submitted vs confirmed.

Here's a thought: maybe we should just get rid of the either-returning version. If people want to handle the error... well, that's what `catchError` is for! That then simplifies our naming scheme, and I think we could just have `writeTx` (although the more I think about it the more I think it should maybe be `submitTx`) and `writeTxConfirmed`.

WDYT?

Reply:
yeah that makes sense. I'll change it to `submitTx` as well.